src/SDKs/Authorization/Authorization.Tests/Tests/BasicTests.cs

```diff
@@ -24,7 +24,8 @@ namespace Authorization.Tests
 {
     public class BasicTests : TestBase, IClassFixture<TestExecutionContext>
     {
-        private readonly ITestOutputHelper _output;
+		public const string ResourceGroup = "resourcegroups/AzureAuthzSDK";
```

Comment:
this has been made a constant so that in the cleanup code in the TestFixture.cs we wont cleanup all the roleassignments and cleanup only those which come under this resourcegroup